### PR TITLE
Update setup in README to current version

### PR DESCRIPTION
**Pull request by da-kami**
Updates `Build & Run` section. 
Related fix for the `default.toml` of `btsieve`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +37,16 @@ Contains crates specific to our application. Can depend on libraries located in
...
-3. `./target/release/comit_node`
-4. TODO: similar documentation for `btsieve`
+2. Put a [`default.toml`](application/comit_node/config/default.toml) config file into `~/.config/comit_node` or set `COMIT_NODE_CONFIG_PATH` as folder path to where the `default.toml` is located.
+3. Put a [`default.toml`](application/comit_node/config/default.toml) config file into `~/.config/btsieve` or set `BTSIEVE_CONFIG_PATH` as folder path to where the `default.toml` is located
+4. startup bitcoin node (port to be set according to btsieve configuration)
+5. startup ethereum node (port to be set according to btsieve configuration)
+6. startup btsieve: `cargo run --bin btsieve`
+7. startup comit_node: `cargo run --bin comit_node`
 
+Once the comit node was started the user interface is served as specified in the comit_node configuration file, default is `localhost:8008`.
```
Comment (D4nte) [inline README.md]: 🙃 maybe mention that this happens if the `web_gui` section is present in the config file?

Comment (thomaseizinger) [reply README.md]: `8008`? what is that? :D

Comment (D4nte) [reply README.md]: > `8008`? what is that? :D

I assume it's a typo

Comment (da-kami) [reply README.md]: I adapted the sentence and fixed the typo, please take another look and approve.

File: README.md [hunk 2]
```diff
@@ -37,10 +37,16 @@ Contains crates specific to our application. Can depend on libraries located in
 ## Build & Run
 
 1. `cargo build` (do `export SOLC_BIN=/usr/bin/solc` if `solc` is installed locally)
```
Comment (bonomat) [inline README.md]: the second part should be outdated after #951 is merged

File: README.md [hunk 3]
```diff
@@ -37,10 +37,16 @@ Contains crates specific to our application. Can depend on libraries located in
...
-2. Put a [`default.toml`](application/comit_node/config/default.toml) config file into `~/.config/comit_node` or set `COMIT_NODE_CONFIG_PATH` to wherever the config file is located.
-3. `./target/release/comit_node`
-4. TODO: similar documentation for `btsieve`
+2. Put a [`default.toml`](application/comit_node/config/default.toml) config file into `~/.config/comit_node` or set `COMIT_NODE_CONFIG_PATH` as folder path to where the `default.toml` is located.
+3. Put a [`default.toml`](application/comit_node/config/default.toml) config file into `~/.config/btsieve` or set `BTSIEVE_CONFIG_PATH` as folder path to where the `default.toml` is located
```
Comment (bonomat) [inline README.md]:
```suggestion
3. Put a [`default.toml`](application/btsieve/config/default.toml) config file into `~/.config/btsieve` or set `BTSIEVE_CONFIG_PATH` as folder path to where the `default.toml` is located
```

File: README.md [hunk 4]
```diff
@@ -37,10 +37,16 @@ Contains crates specific to our application. Can depend on libraries located in
...
+4. startup bitcoin node (port to be set according to btsieve configuration)
+5. startup ethereum node (port to be set according to btsieve configuration)
+6. startup btsieve: `cargo run --bin btsieve`
+7. startup comit_node: `cargo run --bin comit_node`
 
+If the `[web_gui]` section is specified in the configuration the current release of the user interface [comit-i](https://github.com/comit-network/comit-i) will be served once the commit node started up (served at `localhost:8080` as default).
```
Comment (D4nte) [inline README.md]:
```suggestion
If the `[web_gui]` section is specified in the configuration the current release of the user interface [comit-i](https://github.com/comit-network/comit-i) will be served once the comit node started up (served at `localhost:8080` as default).
```

File: README.md (hunk 2 above)
Comment (da-kami) [reply README.md]: Did this suggestion in https://github.com/comit-network/comit-rs/pull/951 as it is related to that task. 
Details: https://github.com/comit-network/comit-rs/pull/951/commits/10c312bdddd1a3f6e3366b3b4ee076db47fd2fbc

**Pull request merged**